frontend/src/features/mapSideBar/SidebarContents/subforms/ValuationForm.scss

```diff
@@ -1,7 +1,35 @@
 .val-table {
-  justify-content: center;
   .table {
     align-content: center;
-    width: 800px;
+    &.netbook {
+      width: 600px;
+      .tbody {
+        max-height: 200px;
+        overflow-y: auto;
+      }
+    }
+    &.assessed {
+      .tbody {
+        max-height: 150px;
+        overflow-y: auto;
+      }
+      width: 520px;
+    }
+    .form-group {
+      margin-bottom: 0;
+    }
+    .thead :first-child .th {
+      justify-content: flex-start !important;
+    }
+    .td,
+    .th {
+      border: 2px solid #dee2e6;
+      justify-content: center !important;
```

Comment:
I don't like this but a whole bunch of !importants were used in the original Table.scss to get things working.